Configuration/EventContent/python/AlCaRecoOutput_cff.py

```diff
@@ -38,6 +38,9 @@
 ###############################################################
 # ECAL calibration with isol. electrons
 from Calibration.EcalAlCaRecoProducers.ALCARECOEcalCalIsolElectron_Output_cff import *
+from Calibration.EcalAlCaRecoProducers.ALCARECOEcalUncalIsolElectron_Output_cff import *
+from Calibration.EcalAlCaRecoProducers.ALCARECOEcalRecalIsolElectron_Output_cff import *
+
```

Comment:
Hi @nilina why do these lines (unrelated to your PR appear here)? Have you started from the last 74X IB?